Sprint-2/extend/format-time.js

```diff
@@ -0,0 +1,29 @@
+// This is the latest solution to the problem from the prep.
+// Your task is to write tests for as many different groups of input data or edge cases as you can, and fix any bugs you find.
+
+function formatAs12HourClock(time) {
+  const hours = Number(time.slice(0, 2));
+  const minutes = time.slice(3);
+
+  if (hours === 0) return `${12}:${minutes} am`;
```

Comment:
Code is correct but why not just typed `12` (instead of ${12}`? 😆

Reply:
I want to to show 12:00 am and minutes same as well

Reply:
I mean, why not write 
```
`${12}:${minutes} am`
```
as
```
`12:${minutes} am`
```
Same as the pm one.